.gitignore

```diff
@@ -4,3 +4,5 @@ node_modules/
 private-key.pem
 .env
 package-lock.json
+.idea/
+probot-settings.iml
```

Comment:
I'd prefer not to add these to the project. See https://help.github.com/articles/ignoring-files/#create-a-global-gitignore for creating a local gitignore file for ignoring editor-specific files.